media/css/mozorg/contact-spaces.less

```diff
@@ -437,6 +437,11 @@
     }
 }
 
+/* override float rule for contact us tab */
+.entry.contact-us {
+    float: none;
```

Comment:
Did not run into any floating issues without the mixin so, perhaps it is not required?